pkg/gitreceive/build.go

```
@@ -265,3 +247,15 @@ func build(conf *Config, s3Client *s3.S3, builderKey, rawGitSha string) error {
 
 	return nil
 }
+
+func prettyPrintJSON(data interface{}) string {
+	output := &bytes.Buffer{}
+	if err := json.NewEncoder(output).Encode(data); err != nil {
+		return ""
```

This should return the error instead of suppressing it